app/src/main/java/com/google/developer/udacityalumni/adapter/ArticleAdapter.java

```diff
@@ -58,6 +60,14 @@ public ArticleViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(ArticleViewHolder holder, int position) {
         mCursor.moveToPosition(position);
+        if (position > mPreviousPosition) {
+            AnimationUtils.scaleXY(holder);
```

Comment:
As both branches of the `if/else` are identical, it's possible to remove it (and the `mPreviousPosition` variable) and replace it with just `AnimationUtils.scaleXY(holder);`